cloud/awsprovider.go

```diff
@@ -506,6 +508,17 @@ func (aws *AWS) DownloadPricingData() error {
 		pvkeys[key.Features()] = key
 	}
 
+	reserved, err := aws.getReservedInstances()
+	if err != nil {
+		klog.V(1).Infof("Failed to lookup reserved instance data: %s", err.Error())
```

Comment:
Yeah, since this is happening just once every DownloadPricing(), then it's probably good to have visibility into the "failure." Otherwise, if it's verbose and noisey, keep it out of prod logs. At least that's my perspective